modules/ROOT/pages/values-and-types/converting-data.adoc

```diff
@@ -0,0 +1,134 @@
+:description: This section provides information about how to convert data values using Cypher functions.
+[[converting-data-values]]
```

Comment:
I do believe casting is the correct term here, but I am not 100% certain, it may be useful to ask in the CLG channel about this :)